app/serializers/document_serializer.rb

```diff
@@ -15,7 +15,7 @@ def filename
   end
 
   def filesize
-    number_to_human_size(object.file.byte_size, precision: 2)
+    object.file.byte_size
```

Comment:
Have you checked with @teneightfive that this change from `96 KB` to a byte count doesn't have an impact on the frontend? If so, :+1:

Reply:
he suggested it

Reply:
could this transform be handled in the front end instead? that seems a more obvious place to do it ?

Reply:
sorry ignore my comment, haven't had ☕️